ADRs/0003-llm-benchmark-configuration.md

```diff
@@ -0,0 +1,25 @@
+# LLM-0003 LLM model configuration
+
+## Context
+
+In order to benchmark LLM models we need to standardize the configuration of running the benchmarks against the model.
+To do this effectively we need to standardize the benchmarks configuration.
+
+## Assumptions
+
+* We run the LLM in kubernetes
+* Benchmarks are stored in git
```

Comment:
Would we want to specify LFS here as well? As for data like this, it is commonly not stored raw in Git

Reply:
I do not think the benchmarks will use LFS

Reply:
The number

Reply:
Allright I agree with you, my train of thought was that for data (read .csv or .json) git is no place, but having version on these benchmarks can be valuable.